### PR TITLE
Added description to configuration properties in spec.yaml (PORT-9538)

### PR DESCRIPTION
**Pull request by mk-armah**
# Description

- Added description to configuration properties in spec.yaml (PORT-9538)
-  Added _target='blank' attribute to specification links to ensure they open in a new browser tab instead of the current browser tab.

## Type of change

Please leave one option from the following and delete the rest:
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

**Review by PeyGis — APPROVED** [review]
LGTM

File: integrations/gitlab/.port/spec.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ configurations:
   - name: tokenMapping
     required: true
     type: object
-    description: "Mapping of Gitlab tokens to the groups scopes to ingest data from into port. Example: {\"THE_GROUP_TOKEN\":[\"getport-labs/**\", \"GROUP/PROJECT PATTERN TO RUN FOR\"]}"
+    description: 'Mapping of Gitlab tokens to the groups scopes to ingest data from into port. Example: {\"THE_GROUP_TOKEN\":[\"getport-labs/**\", \"GROUP/PROJECT PATTERN TO RUN FOR\"]}. To create a group token, see the <a target="_blank" href="https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#create-a-group-access-token-using-ui">Gitlab Documentation</a>'
```
Comment (Tankilevitch) [inline integrations/gitlab/.port/spec.yaml:15]: redundant `'`

Comment (mk-armah) [reply integrations/gitlab/.port/spec.yaml:15]: image

Comment (mk-armah) [reply integrations/gitlab/.port/spec.yaml:15]: with quote

image

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**